tinkerer-materials/quickstarters.md

```diff
@@ -0,0 +1,65 @@
+## Hackathon Quickstarters
+
+Whether you're creating web applications, CLI tools, analytics dashboards, or entirely new blockchains, this section aims to provide the essential SDKs, templates, and examples to get you started quickly.
+
+- **TypeScript dApps** - Build web applications with libraries like PAPI and DeDot.
+- **Rust Applications** - Create high-performance tools and backends with `subxt`, or custom blockchain runtimes with the `polkadot-sdk`
+- **Data & Analytics** - Access blockchain data through REST APIs like Substrate Sidecar and build powerful dashboards with tools like Dotlake.
+- **Ready-to-use Templates** - Boilerplates and working examples to jumpstart development for the above!
+- **Quick Start Guides** - Get a basic Polkadot-based project up and running in minutes with step-by-step instructions
```

Comment:
Make these anchor tags to the below content and I would make "PAPI", "DeDot", "subxt", etc all links.